english/super-man/save-the-word.md

```diff
@@ -0,0 +1,12 @@
+After that, suddenly!!! You start to grow up, Big, GIANT ~~~ Also you noticed that a red shirt is now wrapped on your muscular body, and the trangle underwear is mounted on it, shining a great "V" -- the signature of SUPER MAN.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.